lighthouse/hub-controller.md

```diff
@@ -0,0 +1,40 @@
+# Hub Controller for Lighthouse
+
+## Summary
+
+The current Lighthouse implementation has a passive broker, it is a central apiserver that is
+accessed by each cluster in the clusterset. Clusters synchronise resources to and from the
+broker with all sync and reconciliation logic implemented in the clusters.
+
+There are feature gaps that would be much easier to address if we introduce an active broker
+that can implement reconciliation logic.
```

Comment:
Passive broker is part of the broader Submariner architecture which emphasizes distributed clusters where each cluster is autonomous, and the broker is simply some mechanism to disseminate the data which is very simple (could've been a simple etcd service and not K8s API).

If we're going to change that, then it should be discussed on a higher level architecturally and not just in the context of Lighthouse.